packages/eslint-config-udemy-website/index.js

```diff
@@ -229,12 +229,12 @@ module.exports = {
             'error',
             [
                 {
-                    cdn: 'udemy-images.udemy.com',
-                    fixWith: 'udLink.toS3Images()',
+                    cdn: 's.udemycdn.com',
+                    fixWith: 'udLink.toStorageStaticAsset()',
                 },
                 {
-                    cdn: 's3.amazonaws.com/udemy-images',
-                    fixWith: 'udLink.toS3Images()',
+                    cdn: 'udemy-prod-static-assets.s3.amazonaws.com',
```

Comment:
wonder if we can just use `udemy-prod-static-assets` here, that would catch someone trying to write s3.amazonaws.com/udemy-prod-static-assets as well

Reply:
Updated it and added a test to check for that format as well